Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM golang:1.14.4 as builder
 WORKDIR /go/src/sigs.k8s.io/gcp-filestore-csi-driver
 ADD . .
 ARG TAG
-RUN CGO_ENABLED=0 go build -a -ldflags '-X main.version='"${TAG:-latest}"' -extldflags "-static"' -o bin/gcp-filestore-csi-driver ./cmd
+RUN CGO_ENABLED=0 go build -a -mod=vendor -ldflags '-X main.version='"${TAG:-latest}"' -extldflags "-static"' -o bin/gcp-filestore-csi-driver ./cmd
```

Comment:
You could run make from the Makefile so the build commands aren't duplicated

Reply:
Ack